base/musl/Dockerfile

```diff
@@ -45,7 +46,7 @@ COPY install-functions.sh .
 COPY backport-epics-base-musl.patch .
 COPY epics-base-static-linking.patch .
 COPY install_epics.sh .
-RUN ./install_epics.sh
+RUN COMMANDLINE_LIBRARY=READLINE_NCURSES ./install_epics.sh
```

Comment:
So it turns out that we rely on [`__has_include` compiler feature to properly link to readline][1] instead of EPICS' implementation in the Debian image. This is not an option here though only because we need to also specify its dependency. That's what I understand about the problem being solved here.

Since older versions of libreadline required also linking directly to ncurses (?), [EPICS has personalized options][2] for that too, including `READLINE_NCURSES`. Thus, it turns out it just a coincidence that we can explore this to make musl build work. Is that right?

If so, it may be better to make explicit in the commit message that we are taking advantage of a option defined for another purpose, as it may be removed from EPICS base if someone decides to drop support for older Linux distros eventually.

[1]: https://github.com/epics-base/epics-base/blob/052a0c7e19580c18077a6fc5cf9f67ce75e3082b/documentation/RELEASE_NOTES.md#automatic-commandline_library-w-newer-compilers
[2]: https://github.com/epics-base/epics-base/blob/052a0c7e19580c18077a6fc5cf9f67ce75e3082b/configure/os/CONFIG.Common.linuxCommon#L38-L43

Reply:
You understood it correctly. I will update the commit to make what's going on clearer.

Reply:
@henriquesimoes lemme know what you think.

Reply:
Seems nice and clear now. Thanks!